.github/workflows/pull-request-update.yml

```diff
@@ -30,27 +30,33 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
       - name: Set up Docker Buildx
         id: buildx
         uses: docker/setup-buildx-action@v2
-      - name: Build prod image
+      - name: Build and push PR image
```

Comment:
can/should we delete images after closing PRs? (and maybe also for *-master pre-release images after an official release) as to clean up the docker hub registry?